node-js-basics-101/node_modules/optimist/example/line_count_wrap.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+var argv = require('optimist')
+    .usage('Count the lines in a file.\nUsage: $0')
+    .wrap(80)
+    .demand('f')
+    .alias('f', [ 'file', 'filename' ])
+    .describe('f',
+        "Load a file. It's pretty important."
+        + " Required even. So you'd better specify it."
```

Comment:
__[jshint]__
* Misleading line break before '+'; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->